components/threading/common/button/button.tsx

```diff
@@ -11,6 +11,7 @@ type Props = {
     append?: ReactNode;
     isActive?: boolean;
     hasDot?: boolean;
+    clamp?: boolean;
```

Comment:
It's not super obvious what `clamp` means in the context of a button -- is there a better name we could use? Or do we need a comment here?

Reply:
Maybe `clampText` and a comment?

Reply:
`allowTextOverflow`, at least, now tells me what the intent is here. (Wasn't sure until now ;p)